config/locales/en/errors.yml

```diff
@@ -111,3 +111,11 @@ en:
           attributes:
             add_offence:
               inclusion: Select yes if you want to add another offence
+        steps/case/offence_date_fieldset_form:
+          attributes:
+            date:
+              future_not_allowed: Offence date cannot be in the future
```

Comment:
We need to add here also the `blank` otherwise on error it looks like this on the field:

<img width="479" alt="Screenshot 2022-09-21 at 11 17 12" src="https://user-images.githubusercontent.com/687910/191479503-2e1229ee-16ae-402d-98de-b99f8eca4d87.png">

In fact, I would suggest copy and paste all date errors from `en.activemodel.errors.models.steps/client/details_form.attributes.date_of_birth` (tweaking the copy and/or removing the ones that can't trigger) and we are good to go.

It may seem like a lot of repetition but as we want unique and explicit error messaging we can't generalise really.

Reply:
Yep course - I didn't know that it had individual errors for constituent parts of the date 👍